opencaesar/kepler16b-example
# Adds OML Vision support to Kepler16b-example

How To Test:
1. Download OML Vision (ping @pogi7 for details)
2. Run `./gradlew clean`
3. Run `./gradlew owlQuery`
4. Open OML Vision from sidebar
5. Click Objectives -> Should see Table View
![image](https://github.com/opencaesar/kepler16b-example/assets/108915844/c3ef1722-8373-4ee0-8efc-1f29ea411a93)

6. Click Components -> Should see Tree View
![image](https://github.com/opencaesar/kepler16b-example/assets/108915844/36197207-6427-4987-83ca-bf93c2854888)

7. Click Missions -> Should see Diagram View
![image](https://github.com/opencaesar/kepler16b-example/assets/108915844/0b0b10a7-635c-4649-9fd9-02ee33dd18bb)
